### ISSUE TITLE
TypeError: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by LvChengbin** [jlmakes/karma-rollup-preprocessor#44]
I got the error message as follow after upgrading the `karma-rollup-preprocessor` to the latest version:

```sh
TypeError: Cannot read property 'forEach' of undefined
    at Watch.capture (.../node_modules/karma-rollup-preprocessor/lib/index.js:65:20)
    at rollup.then.bundle (.../node_modules/karma-rollup-preprocessor/lib/index.js:22:23)
```

The output of printing `bundle` at `lib/index.js:22:23` is(I removed some similar lines and replaced some full paths with '...'):

```sh
{ cache:
   { modules:
      [ [Object],
        [Object],
        [Object],
        ...
      ],
     plugins: {} },
  watchFiles:
   [ '/Users/.../test/basic.spec.js',
     '/Users/.../src/is.js',
     '/Users/.../src/arrow-function.js',
    ],
  generate: [Function: generate],
  write: [Function: write] 
}
```

**Comment by jlmakes** [comment]
Hi @LvChengbin,

Can you be specific: what version of `karma` `karma-rollup-preprcessor` and `rollup` are you using?

**Comment by jlmakes** [comment]
Glad to hear it! The `next` tag will be released as `7.0.0` very soon! (I’ll ping you when it releases)

**Comment by eKoopmans** [comment]
I was receiving a similar error, using the latest `karma-rollup-preprocessor` but with a pre-1.0 version of `rollup`:

```
TypeError: bundle.watchFiles is not iterable
    at preprocess (node_modules\karma-rollup-pre-processor\lib\index.js:33:49)
    at <anonymous>
```

Updating to the latest `rollup` fixed the issue!
